src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -67,11 +65,7 @@ export default function EnableBiometry({ navigation }: Props) {
     navigation.setOptions({
```

Comment:
The back arrow and skip button need to have their colors changed

Reply:
whoops. thanks